### PR TITLE
add a contact form and finish the mobile version

### PR DESCRIPTION
**Pull request by BereketRetta**
This pull request is finishing add a contact form and finish the mobile version with pair peogramming

### CODE REVIEW

File: mobile.css [hunk 1]
```diff
@@ -646,7 +646,7 @@ span {
 .error {
```
Comment (carlos-ssh) [inline mobile.css:646]: **Kindly work on this feature:**
_Provide requested style by the style-guide(Figma)._
- Give style to your project and try to keep straight at the design requirements.

Screen Shot 2021-12-07 at 7 50 34

File: mobile.css [hunk 2]
```diff
@@ -646,7 +646,7 @@ span {
...
   color: #da2c2c;
   font-weight: 400;
-  font-size: 1rem;
+  font-size: 1.6rem;
```
Comment (carlos-ssh) [inline mobile.css:649]: - Update your `README` file.

**Review by carlos-ssh — CHANGES_REQUESTED** [review]
# 🦺 🚧 🚥 CHANGES REQUIRED 🚥 🚧 🦺
#

Hi @BereketRetta , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._ 


changesrequested
## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: mobile.css (hunk 1 above)
Comment (BereketRetta) [reply mobile.css:646]: Hi @carlos-ssh I did the mobile version. so if you can kindly check in your chrome dev tools for mobile you will see that I fulfilled the requirements

File: mobile.css (hunk 2 above)
Comment (BereketRetta) [reply mobile.css:649]: @carlos-ssh what in particular should I update?

Comment (carlos-ssh) [reply mobile.css:649]: You can get a copy provided by Microverse [here](https://github.com/microverseinc/readme-template).

Try to fill all the best.. Remember that this is the first impression on each project. looking to impress recruiters or people without technical knowledge. 

Remember that you can reutilize your `README` for your next projects just changing simple things.

**Review by rotimiazeez — APPROVED** [review]
# Hi Team, 

### Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**